AFQ/utils/volume.py

```diff
@@ -116,7 +119,16 @@ def dice_coeff(arr1, arr2):
     if isinstance(arr2, nib.Nifti1Image):
         arr2 = arr2.get_fdata()
 
-    # scipy's dice function returns the dice *dissimilarity*
-    return 1 - dice(
-        arr1.flatten().astype(bool),
-        arr2.flatten().astype(bool))
+    arr1 = arr1.flatten()
+    arr2 = arr2.flatten()
+
+    if weighted:
+        return (
+            np.sum(arr1 * arr2.astype(bool))
+            + np.sum(arr2 * arr1.astype(bool)))\
```

Comment:
I don't think you need this slash because the statement is in parentheses

```suggestion
            + np.sum(arr2 * arr1.astype(bool)))
```

Reply:
The backslash here is necessary because those two statements are in different parentheses